util/src/test/java/io/grpc/util/OutlierDetectionLoadBalancerTest.java

```diff
@@ -193,6 +198,11 @@ public Void answer(InvocationOnMock invocation) throws Throwable {
           }
         });
 
+    // when(mockStreamTracerFactory.newClientStreamTracer(isA(StreamInfo.class),
+    //     isA(Metadata.class))).thenReturn(mockStreamTracer);
```

Comment:
Any reason to keep the commented code around?

Reply:
No, just an oversight - removed now.